extensions/wikia/TemplateClassification/scripts/TemplateClassificationLabeling.js

```diff
@@ -52,39 +52,30 @@ define('TemplateClassificationLabeling',
 		})();
 
 		getTitleStrategy = (function getTitleStrategy() {
-			function addTemplate() {
-				return mw.message('template-classification-edit-modal-title-add-template').escaped();
-			}
-
 			function editType() {
 				return mw.message('template-classification-edit-modal-title-edit-type').escaped();
 			}
 
-			function addTypeBeforePublish() {
-				return mw.message('template-classification-edit-modal-select-type-sub-title').escaped();
+			function chooseType() {
+				return mw.message('template-classification-edit-modal-title-select-type').escaped();
 			}
 
 			return {
-				addTemplate: addTemplate,
+				addTemplate: chooseType,
 				editType: editType,
-				addTypeBeforePublish: addTypeBeforePublish
+				addTypeBeforePublish: chooseType
```

Comment:
Maybe I misworded. Whole point of Labeling ext here is to provide modes that TC modal can pick. Please lookup var availableModes https://github.com/Wikia/app/blob/2d2ed7cd1628767bfac087845f99afd3be37a09f/extensions/wikia/TemplateClassification/scripts/TemplateClassificationLabeling.js
I consider this naming suits best. It explains what the mode is about so you can setup different labels for title button and content. Please note that combinations of labels are different for each mode so we can't merge any of them right now.
I'm also open for any naming improvements if you have a better idea or what do you think about moving on with this.